services/user/CommonSys/ui/src/App.tsx

```diff
@@ -1,19 +1,22 @@
-import { siblingUrl } from "@psibase/common-lib";
+import { Nav } from "@/components/nav";
+import { AppItem } from "@/components/app-item";
 
 const App = () => {
     return (
-        <div className="flex h-screen w-screen flex-col">
-            Under construction...
-            <ul>
-                <li>
-                    <a href={siblingUrl(null, "explore-sys", null, false)}>
-                        Explorer
-                    </a>
-                </li>
-                <li>
-                    <a href={siblingUrl(null, "doc-sys", null, false)}>Docs</a>
-                </li>
-            </ul>
+        <div className="mx-auto h-screen w-screen max-w-screen-lg">
+            <Nav />
+            <div className="flex flex-col sm:flex-row">
+                <AppItem
+                    name="Explorer"
+                    description="Explore recent transactions and chain history."
+                    service="explore"
+                />
+                <AppItem
+                    name="Doc"
+                    description="Review blockchain documentation."
+                    service="doc"
```

Comment:
The service is currently "doc-sys", but after my [pr](https://github.com/gofractally/psibase/pull/677) it will actually be, "docs"